src/Type/Symfony/InputBagDynamicReturnTypeExtension.php

```diff
@@ -70,18 +70,11 @@ private function getAllTypeFromMethodCall(
 		MethodCall $methodCall
 	): Type
 	{
-		$types = [
-			new BooleanType(),
-			new FloatType(),
-			new IntegerType(),
-			new StringType(),
-		];
-		$oneParameterType = new UnionType($types);
 		if (isset($methodCall->getArgs()[0])) {
-			return new ArrayType(new MixedType(), $oneParameterType);
+			return new ArrayType(new MixedType(), new MixedType());
```

Comment:
Can you use `new MixedType(true)` so that it causes issues at least on level 9?

Reply:
@ondrejmirtes Changed it